Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs

```diff
@@ -158,6 +158,11 @@ private object MapActivityToLiteDb(Activity activity, Context context, eExecuted
             }
             activity.Acts.ToList().ForEach(action => this.MapActionToLiteDb((GingerCore.Actions.Act)action, context, executedFrom));
             AR.ActionsColl.AddRange(liteDbActionList);
+            //counts
+            AR.ChildExecutableItemsCount.Add("Action", activity.Acts.Count);
```

Comment:
for all levels ChildExecutableItemsCount calculations- you need to filter the non Active items, count only Active items.